source/docs/software/vision-processing/wpilibpi/installing-the-image-to-your-microsd-card.rst

```diff
@@ -19,8 +19,8 @@ of the list of releases.
 Copy the image to your MicroSD card
 -----------------------------------
 
-Download and install Etcher (https://www.balena.io/etcher/) to image the micro SD card. The micro SD card needs to be at least
-4 GB. Note: a micro SD to USB dongle such as https://www.amazon.com/gp/product/B0779V61XB works well for writing to micro SD cards.
+Download and install `Etcher <https://www.balena.io/etcher/>`__ to image the micro SD card. The micro SD card needs to be at least
+8 GB. Note: a `micro SD to USB dongle <https://www.amazon.com/gp/product/B0779V61XB>`__ works well for writing to micro SD cards.
```

Comment:
```suggestion
Download and install `Etcher <https://www.balena.io/etcher/>`__ to image the micro SD card. The micro SD card needs to be at least 8 GB. A `micro SD to USB dongle <https://www.amazon.com/gp/product/B0779V61XB>`__ works well for writing to micro SD cards.
```

Note should either be an actual ``.. note:: `` or just removed. I don't think it adds much to the sentence.